### ISSUE TITLE
numpy 1.22 support

### ISSUE THREAD

**Issue by Kushargs** [mhostetter/galois#368]
Currently, galois depends on NumPy (>=1.18.4, <1.22). Probably because Numba 0.55.1 has a dependency on Numpy with versions < 1.22. But, Numba 0.55.2 now supports NumPy 1.22. So, galois dependency can also be bumped. It will help resolve some vulnerabilities. check this [issue](https://github.com/numba/numba/issues/8025#issue-1227062520).

**Comment by mhostetter** [comment]
Thanks for opening this issue. A contributor recently submitted PR #366 to address this issue. I merged it onto `master`, however I haven't released a new version of `galois` since.

You can install the latest code from `master` like this:

```
$ python3 -m pip uninstall galois
$ python3 -m pip install git+https://github.com/mhostetter/galois.git
```

**Comment by Kushargs** [comment]
My bad, I didn't check properly. Thank you for the help!

**Comment by mhostetter** [comment]
No worries!

**Comment by mhostetter** [comment]
FWIW, this was just released publicly in v0.0.30.
